lte/gateway/c/oai/tasks/grpc_service/AmfServiceImpl.h

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2020 The Magma Authors.
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <grpc++/grpc++.h>
+#include <grpcpp/impl/codegen/status.h>
+
+#include "lte/protos/session_manager.grpc.pb.h"
+#include "lte/protos/spgw_service.grpc.pb.h"
+#include "lte/protos/policydb.pb.h"
+
+extern "C" {
+#include "spgw_service_handler.h"
+#include "log.h"
```

Comment:
We don't need these two headers here, they are only relevant in the .cpp source.

Reply:
Sure, I'll test and push with changes.